src/calculator/Calculator.java

```diff
@@ -93,6 +89,18 @@ public Calculator() {
 		this.setSize(620, 900);
 	}
 
+	private JButton createButton(String name) {
+		JButton button = new JButton(name);
+		button.setFocusable(false);
+		button.setFont(this.buttonFont);
+		button.setBackground(this.buttonColor);
+		button.setBorder(null);
+		button.setForeground(Color.BLACK);
+		button.addActionListener(this);
+		button.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
+		return button;
+	}
```

Comment:
Here I've moved some code into a separate function. Small functions are better than big functions in terms of code maintenance.